setup.py

```diff
@@ -1,14 +1,13 @@
 import os
 
+from pip.download import PipSession
```

Comment:
Any reason for the move?

Reply:
Yes, all imports needs to be before variables.